test/integration/report_test.rb

```diff
@@ -160,6 +160,20 @@ def setup
     end
   end
 
+  test 'successful report with UNIX timestamped transactions' do
+    Timecop.freeze(Time.utc(2010, 4, 23, 00, 00)) do
+      post '/transactions.xml',
+        :provider_key => @provider_key,
+        :transactions => {0 => {:app_id    => @application.id,
+                                :usage     => {'hits' => 1},
+                                :timestamp => '1271980765'}}
```

Comment:
Would be good to add a comment with the date.
`:timestamp => '1271980765'}} # 22/04/2010 @ 11:59pm`